setup.py

```diff
@@ -39,7 +40,7 @@
     license='BSD',
     packages=find_packages(),
     install_requires=[
-        'django>=3.2,<4.2',
+        'django>=3.2,<4.3',
```

Comment:
I think the condition here should be `'django>=3.2,<5.0'` instead `'django>=3.2,<4.3'`. Django always have 2 minor releases after a major release.